apps/server/src/modules/h5p-editor/h5p-editor.module.ts

```diff
@@ -65,5 +65,7 @@ const providers = [
 	imports,
 	controllers,
 	providers,
+	exports: [ContentStorage, LibraryStorage],
 })
 export class H5PEditorModule {}
+export { LibraryStorage, ContentStorage, s3ConfigLibraries, s3ConfigContent };
```

Comment:
Can you please remove this export. All this can import directly from source. The export in this file can run into dependency cycles.

Reply:
solved: 800fd7e85b27fbe9a87d8642054d9ea1ba2d0679